util/version/version_test.go

```diff
@@ -23,6 +23,14 @@ import (
 	. "github.com/onsi/gomega"
 )
 
+func mustParseTolerant(s string) semver.Version {
```

Comment:
Last nit, I promise :).

Can you please move this func to the bottom of the file?

Just in general we roughly try to topology-sort funcs.
We're not really doing this consistently, but I think it would be good to have helper funcs at the bottom.

Otherwise lgtm from my side.

Reply:
No problem, I will move it to the bottom of the file.